experimental/material_3_demo/lib/main.dart

```diff
@@ -3,14 +3,18 @@
 // found in the LICENSE file.
 
 import 'package:flutter/material.dart';
+import 'package:flutter/scheduler.dart';
+
 import 'color_palettes_screen.dart';
 import 'component_screen.dart';
 import 'elevation_screen.dart';
 import 'typography_screen.dart';
 
 void main() {
   runApp(const MaterialApp(
-      debugShowCheckedModeBanner: false, home: Material3Demo()));
+    debugShowCheckedModeBanner: false,
+    home: Material3Demo(),
+  ));
```

Comment:
If you've added a trailing comma, I'd add another on L17?

Reply:
done!